cli/Application/Command/Get/Project/Issues.php

```diff
@@ -47,6 +47,15 @@ class Issues extends Project
 	protected $issues = array();
 
 	/**
+	 * Status of issues.
+	 *
+	 * @var array
+	 *
+	 * @since  1.0
+	 */
+	protected $issuesStatus = array();
```

Comment:
Could you name this `$issueStatuses` or even better `$issueStates`?
Arrays always should be named as plurals :wink: 

We are also using the "new" array syntax `[]`here . We aren't consistent yet, but new code should use the new syntax :stuck_out_tongue_winking_eye: 

Finally you can initialize the array here instead of line #`121`, so I'd propose:

`protected $issueStates = ['open', 'closed'];`

How does that look (and feel)?